tmt-app/src/Components/CustomCard.jsx

```diff
@@ -0,0 +1,124 @@
+import React, { Fragment } from "react";
+import {
+  Card,
+  CardContent,
+  CardMedia,
+  Typography,
+  Container,
+} from "@mui/material";
+import { ReactComponent as EmotionalIntelligence } from "../Images/icons/emotional_intelligence.svg";
+import { Link } from "react-router-dom";
+
+function CustomCard({ dummyData }) {
+  return (
+    <Fragment>
+      {dummyData.map((item) => {
+        return (
+          <Link
+            rel="noopener noreferrer"
+            to={`/career/${item.name}`}
+            style={{
+              textDecoration: "none",
+            }}
+            key={item.id}
+          >
+            <Card
+              sx={{
+                display: "flex",
+                alignItems: "center",
+                backgroundColor: "Matcha.main",
+                borderRadius: "15px",
+                color: "white",
+                ":hover": {
+                  color: "black",
+                  backgroundColor: "BTMilkTea.main",
+                },
+                cursor: "pointer",
+              }}
+            >
+              <CardMedia
+                component="img"
+                image={item.icon}
```

Comment:
If we use the `component` prop to assign the `img` element, does MUI preserve access to this `image` prop still?